src/service/performance-impl.js

```diff
@@ -130,10 +172,11 @@ export class Performance {
     this.supportsLayoutShift_ = supportedEntryTypes.includes('layout-shift');
 
     if (!this.supportsLayoutShift_) {
-      this.metrics_.rejectSignal(
-        TickLabel.CUMULATIVE_LAYOUT_SHIFT,
-        dev().createExpectedError('Cumulative Layout Shift not supported')
+      const e = dev().createExpectedError(
```

Comment:
Since you're here, consider `devExpectedError` from `#core/error`

Reply:
Since there are so many other calls, I'm going to defer.